### PR TITLE
Storybook components

### PR DESCRIPTION
**Pull request by caiokf**
Adding stories for other components on Storybook. Covered in this PR:
- [x] Text
- [x] Heading
- [x] Icon
- [x] EventCard

### CODE REVIEW

File: src/components/Text/index.tsx [hunk 1]
```diff
@@ -36,6 +36,13 @@ export const uppercase = props =>
       }
     : null
 
+export const capitalize = props =>
```
Comment (drydenwilliams) [inline src/components/Text/index.tsx]: 🤔

Comment (BenGamma) [reply src/components/Text/index.tsx]: Yes some components needs refactoring, especially `Text` I've addressed the issue here #727

Comment (caiokf) [reply src/components/Text/index.tsx]: @drydenwilliams 
Check this [commit](https://github.com/ONEARMY/community-platform/pull/769/commits/b11307afab0cc22d6231eeb07ae506b6bdb57760) and see if this looks a bit better (would look nicer without lint breaking lines, but maybe still ok)

**Comment by caiokf** [comment]
@BenGamma Please take a look, I managed to include a few more components, although not a comprehensive list yet. There are a few commits that touch some of the component's code, so please check those too.

**Comment by BenGamma** [comment]
Storybook works like a charm thanks !
But when I'm running the basic `yarn start` I have a few errors on compile :
- First with a `onClick` props to change to `OnClick`, which is quite straightforward.
- The second is about typings (see screenshot or run `yarn start` to reproduce). Can you resolve before I merge ? Thanks
Capture d’écran 2019-11-15 à 13 10 25

**Comment by caiokf** [comment]
@BenGamma Yes, will fix it. But question about the `OnClick`: is there a reason why it is capitalized? it comes from this [commit](https://github.com/ONEARMY/community-platform/commit/b67fc708eb0463202ea01678ce7dcb25a2b01bd0). Can I make it lowercase everywhere?

> Storybook works like a charm thanks !
> But when I'm running `yarn start` I have a few errors on compile :
> 
> * First with a `onClick` props to change to `OnClick`, which is quite straightforward.
> * The second is about typings (see screenshot or run `yarn start` to reproduce). Can you resolve before I merge ? Thanks
> 
> Capture d’écran 2019-11-15 à 13 10 25

**Comment by BenGamma** [comment]
> @BenGamma Yes, will fix it. But question about the `OnClick`: is there a reason why it is capitalized? it comes from this [commit](https://github.com/ONEARMY/community-platform/commit/b67fc708eb0463202ea01678ce7dcb25a2b01bd0). Can I make it lowercase everywhere?
> 
> > Storybook works like a charm thanks !
> > But when I'm running `yarn start` I have a few errors on compile :
> > 
> > * First with a `onClick` props to change to `OnClick`, which is quite straightforward.
> > * The second is about typings (see screenshot or run `yarn start` to reproduce). Can you resolve before I merge ? Thanks
> > 
> > Capture d’écran 2019-11-15 à 13 10 25

Yes you can lowercase it. It's legacy code, probably a mistake by the time 😇

**Pull request merged**